### ISSUE TITLE
Administrator comments placed in moderation queue 

### ISSUE THREAD

**Issue by ghost** [Automattic/jetpack#33443]
### Impacted plugin

Jetpack

### Quick summary

After a recent update (not quite sure which, since the issue appeared only after logging out and back in), leaving a reply on my own site using the administrator email placed the comment in moderation queue (requiring going to the dashboard to approve my own comment).

Parenthetical remark: in the past month or so, I have also stopped receiving email notifications about new comments. I wonder if there is a connection.

### Steps to reproduce

1. If logged in, log out.
2. Log in using the WordPress account listed as an administrator on the given site.
3. Reply to a comment (likely the same result should happen if writing a new comment).
4. Notice the comment is placed in a moderation queue.

### A clear and concise description of what you expected to happen.

Administrator replies (as was the behavior in the past) should be approved automatically.

### What actually happened

Administrator reply was placed in a moderation queue and had to be approved (by the very same person; approving their own comment)

### Impact

One

### Available workarounds?

No but the platform is still usable

### Platform (Simple and/or Atomic)

Self-hosted

**Comment by jeherve** [comment]
Could you go to Jetpack > Settings > Discussion in your dashboard, and disable the following option:

"Let visitors use a WordPress.com or Facebook account to comment"

Could you then try to leave another comment, and let me know if the problem remains?

Thank you!

> Parenthetical remark: in the past month or so, I have also stopped receiving email notifications about new comments. I wonder if there is a connection.

It may not be connected. Those email notifications are sent directly by your WordPress installation, they do not rely on any plugin.

**Comment by ghost** [comment]
No, disabling the option allowed the comment to be accepted automatically.

**Comment by jeherve** [comment]
Ok, thanks for giving this a try!

Could you let me know:

- What version of Jetpack is currently installed on your site?
- What are your discussion settings (under Settings > Discussion)? Could you make a screenshot of the page and post it here?
- What other plugins are installed on your site and make come into play here? Do you use any anti-spam or commenting plugins?
- When you are about to leave a comment on the site while logged in, could you make a screenshot of the comment form that is displayed on the page, and post it here?

Thank you!

**Comment by ghost** [comment]
>     * What version of Jetpack is currently installed on your site?

12.6.2

>     * What are your discussion settings (under Settings > Discussion)? Could you make a screenshot of the page and post it here?

![8OB6U4zLwbzB](https://github.com/Automattic/jetpack/assets/426388/ceef776b-d2d3-4ad9-983c-654b55ba328a)

(I don't share the settings below for privacy reasons - contains IPs - and also because it shouldn't be relevant)
> 
>     * What other plugins are installed on your site and make come into play here? Do you use any anti-spam or commenting plugins?

Yoast SEO and Classic Editor
> 
>     * When you are about to leave a comment on the site while logged in, could you make a screenshot of the comment form that is displayed on the page, and post it here?

 Well, there is no comment form when I'm logged in; only the textarea and the "Reply" button, with my user icon right below. When not logged  in, it of course displays (once I start typing) the "Log in to comment" prompt, showing the WordPress, Facebook, and Email icons.
 
Another thing I realize only now: I recently disconnected my WordPress from JetPack (i.e. disabled WordPress.com login). Would that be relevant, perhaps? 
*Edit:* Just to clarify, I never logged in with WordPress; I always log in with email. I simply disabled WordPress.com login

**Comment by jeherve** [comment]
Thank you!

> Well, there is no comment form when I'm logged in; only the textarea and the "Reply" button, with my user icon right below.

This is odd. Do you not see any comment form when you're logged in at all? I would expect you to see one of the below:

<img width="779" alt="Screenshot 2023-10-04 at 12 15 05" src="https://github.com/Automattic/jetpack/assets/426388/922968ed-f14c-4ef9-a604-a8fb827c0b63">
<img width="765" alt="Screenshot 2023-10-04 at 12 14 23" src="https://github.com/Automattic/jetpack/assets/426388/5d05b15f-f156-46a6-bfb9-a0e6bdf1a1f7">

> I recently disconnected my WordPress from Jetpack (i.e. disabled WordPress.com login).

Do you mean that you've disabled the option under Jetpack > Settings > Security > WordPress.com login? This option is not needed for the Jetpack comment form option to work, no worries there!

**Comment by ghost** [comment]
> This is odd. Do you not see any comment form when you're logged in at all? I would expect you to see one of the below:

Perhaps  I misunderstood or didn't make myself clear: I see the textarea and the reply button. When I begin to type, I either see my user icon (when logged in) or a prompt to log in (when logged out). See images below for reference:

https://pasteboard.co/ZRV1faKIiSsl.png
https://pasteboard.co/f1w8whBrowoG.png
 
> Do you mean that you've disabled the option under Jetpack > Settings > Security > WordPress.com login? This option is not needed for the Jetpack comment form option to work, no worries there!

That's the one, yes.

**Comment by jeherve** [comment]
Thanks for the extra details!

@Automattic/vertex Since this touches some of the work you've done with Verbum, do you think you could take a look?

Thank you!

**Comment by heavyweight** [comment]
Thanks for the heads up!! We are taking a look.

**Comment by heavyweight** [comment]
Switched to highlander to check if I could replicate and it has the correct behavior ✅

**Comment by heavyweight** [comment]
We were fixing a related issue and to me it seems that this got fixed as well.
@LifeIsADeflatingVest and @jeherve can you try to reproduce? No need for any testing setup except a sandbox pull.

**Comment by jeherve** [comment]
This tested well for me just now. @LifeIsADeflatingVest Could you give it a try on your site, and let us know if you still experience issues?

**Comment by ghost** [comment]
Works on my end too. Closing this, thanks for the help.
